apps/src/templates/projects/FeaturedProjectsTable.story.jsx

```diff
@@ -1,25 +0,0 @@
-import React from 'react';
```

Comment:
Removing storybook because not a re-usable component.
In progress: `FeaturedProjectsTableTest.js` similar to `PersonalProjectsTableTest.js`.